Core/src/com/infiniteautomation/mango/spring/components/EmailAddressVerificationService.java

```diff
@@ -75,7 +76,12 @@ public EmailAddressVerificationService(
             UsersService usersService,
             PublicUrlService publicUrlService,
             SystemSettingsDao systemSettings,
-            PermissionService permissionService, RunAs runAs, PageResolver pageResolver) {
+            PermissionService permissionService,
+            RunAs runAs,
+            PageResolver pageResolver,
+            Clock clock) {
```

Comment:
Modified API in a patch release, might be safer to not backport the tests (this change was for testing only).

Reply:
Ok, undoing it.